### PR TITLE
Fix install script's storing of config.json values

### LINKED ISSUE #1828: Install script doesn't take into account all config.json values
## Expected Behaviour
Script should use all values in `config.json`, removing the need to input these values again.

## Actual Behaviour
Script prompts the user for inputs from the `inputData.features` inputs onwards.

## Steps to Reproduce
1. Run through install script, choosing `y` when asked to use `config.json` values

## Versions
  - Authoring Tool Version: 0.4.0

### PR DESCRIPTION
**Pull request by taylortom**
Fixes #1828 and #1842.

### CODE REVIEW

**Comment by dancgray** [comment]
Is useSmtp meant to be populated, seems to be missing in my tests.

File: install.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ installHelpers.getLatestFrameworkVersion(function(error, latestFrameworkTag) {
 function generatePromptOverrides() {
   if(USE_CONFIG) {
     var configJson = require('./conf/config.json');
-    var configData = JSON.parse(JSON.stringify(configJson).replace('true', '"y"').replace('false', '"n"'));
+    var configData = JSON.parse(JSON.stringify(configJson).replace(/true/g, '"y"').replace(/false/g, '"n"'));
```
Comment (canstudios-louisem) [inline install.js]: This feels very hacky.

Comment (taylortom) [reply install.js]: This was to deal with how we use y/n for boolean input values (which is the default for prompt). We could maybe look into also accepting true/false as values.

Comment (taylortom) [reply install.js]: I'm happy to look into it now, but as it's an enhancement, I'd prefer to push until 0.5.0 (or even the next patch if we have one)

**Comment by taylortom** [comment]
What's the consensus with this one?

**Pull request merged**